examples/Training/python/Advanced/Create_Image_advanced.py

```diff
@@ -7,14 +7,18 @@
 # Use is subject to license terms supplied in LICENSE.txt
 #
 
+import sys
+import os
+sys.path.append(os.path.join('..', 'python'))
```

Comment:
``from ..Parse_OMERO_Properties import...`` does not work with the actual set-up